packages/11ty/_layouts/objects-catalog.webc

```diff
@@ -0,0 +1,8 @@
+---
+layout: base.11ty.js
+---
+<page-header></page-header>
```

Comment:
see comment above about the `page-header` WebC wrapper component https://github.com/thegetty/quire/pull/835/files#r1322062974